ngraph/core/shape_inference/include/lstm_cell_shape_inference.hpp

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+#include <rnn_cell_shape_inference.hpp>
+#include "shape_infer_utils.hpp"
+
+namespace ov {
+namespace op {
+namespace ShapeInferLSTM {
+template <class T1, class T2>
```

Comment:
lets give them proper names. Smth like OpType and ShapeType?

Reply:
resolved in f7d415276b3f8a48e0b32b1bd918a86d70bf6623